framework/src/postprocessors/ConstantPostprocessor.C

```diff
@@ -25,6 +25,12 @@ ConstantPostprocessor::validParams()
 ConstantPostprocessor::ConstantPostprocessor(const InputParameters & params)
   : GeneralPostprocessor(params), _value(getParam<Real>("value"))
 {
+  const PostprocessorReporterName r_name(name());
+  auto & write_data = _fe_problem.getReporterData(ReporterData::WriteKey());
+
+  // Initialize values
+  const auto value = getParam<Real>("value");
+  write_data.setReporterValue<PostprocessorValue>(r_name, value);
```

Comment:
```suggestion
  write_data.setReporterValue<PostprocessorValue>(r_name, _value);
```

Reply:
if we want the control to be able to mess with it on initial ?

Reply:
`getParam<Real>("value")` represents the same thing as `_value`, a reference to the controllable parameter in the input warehouse. But might as well do the change for cleaner DRY code